src/client/.eslintrc.yml

```diff
@@ -7,7 +7,19 @@ extends:
   - plugin:react/jsx-runtime
   - plugin:@typescript-eslint/recommended
   - prettier
-overrides: []
+overrides:
+  [
+    {
+      files: ["*.{mock,test,spec,stories}.{ts,tsx}", "**/__mocks__/*"],
```

Comment:
If you look at the diffs, they are disabling many of the same rules.

Reply:
And as non-app code, it seems fine to me.

Reply:
it is app code tho .. this is published

Reply:
Well, I think of Storybook as documentation.  Either way, I reverted applying the looser typing rules to story files and amended the types so that we don't have to manually disable eslint rules in those files.